src/api/helpers/formatEntityData.js

```diff
@@ -0,0 +1,386 @@
+			"ended": false,
+			"type": "Publisher"
+		}
+ */
+
+export function getPublisherBasicInfo(publisher: object) {
+	return _.isNil(publisher) ? null :
+		{
+			area: _.get(publisher, 'area.name', null),
+			bbid: _.get(publisher, 'bbid', null),
+			beginDate: _.get(publisher, 'beginDate', null),
+			defaultAlias: getDefaultAlias(publisher),
+			disambiguation: _.get(publisher, 'disambiguation.comment', null),
+			endDate: _.get(publisher, 'endDate', null),
+			ended: _.get(publisher, 'ended', null),
+			type: _.get(publisher, 'publisherType.label', null)
+		};
+}
+
+/**
+ * getEntityAliases is a function to extract the list of aliases from ORM entity
+ *
+ * @param {object} entity - An ORM Entity
+ * @returns {object} an object conatining the bbid and list of aliases data of an ORM entity
+ * Each property of object defaults to null. If entity is null or undefined, returns null.
+ * @example
+ *		const edition = await orm.func.entity.getEntity(orm, 'Edition', bbid, relations);
+ *		getEntityAliases(edition);
+ *		/* => {
+			"aliases": [
+				{
+					"aliasLanguage": "English",
+					"name": "A Monster Calls",
+					"primary": true,
+					"sortName": "Monster Calls, A"
+				}
+			],
+			"bbid": "442ab642-985a-4957-9d61-8a1d9e82de1f"
+		}
+ */
+
+export function getEntityAliases(entity: object) {
+	return _.isNil(entity) ? null :
+		{
+			aliases: _.get(entity, 'aliasSet.aliases', []).map((alias) => ({
+				aliasLanguage: _.get(alias, 'language.name', null),
+				name: _.get(alias, 'name', null),
+				primary: _.get(alias, 'primary', null),
+				sortName: _.get(alias, 'sortName', null)
+			})),
+			bbid: _.get(entity, 'bbid', null)
+		};
+}
+
+/**
+ * getEntityIdentifiers is a function to extract the list of indentifiers from an ORM entity
+ *
+ * @param {object} entity - An ORM entity
+ * @returns {object} an object containing the bbid and list of identifier data of an entity
+ * Each property of object defaults to null. If entity object is null or undefined, returns null.
+ *
+ * @example
+ *		const edition = await orm.func.entity.getEntity(orm, 'Edition', bbid, relations);
+ *		getEntityIdentifiers(edition);
+ *		/* => {
+			"bbid": "442ab642-985a-4957-9d61-8a1d9e82de1f",
+			"identifiers": [
+				{
+					"type": "Amazon ASIN",
+					"value": "B00A9U9J6C"
+				}
+			]
+		}
+ */
+
+export function getEntityIdentifiers(entity: object) {
+	return _.isNil(entity) ? null :
+		{
+			bbid: _.get(entity, 'bbid', null),
+			identifiers: _.get(entity, 'identifierSet.identifiers', []).map((identifier) => ({
+				type: _.get(identifier, 'type.label', null),
+				value: _.get(identifier, 'value', null)
+			}))
+		};
+}
+
+/**
+ * getEntityRelationships is a function to extract the list of relationships from an ORM entity
+ *
+ * @param {object} entity -  An ORM entity
+ * @returns {object} an object containing the bbid and list of ralationship data of an ORM entity
+ * Each property of object defaults to null. If entity is null or undefined, returns null.
```

Comment:
'Each property of object defaults to null' should be replaced with "The relationships property defaults to an empty array"